config/routes.rb

```diff
@@ -141,11 +135,18 @@
         :via => :get
   ####
 
-  #### Citations controller
   scope path: 'request/:url_title' do
+    #### Citations controller
     resources :citations, only: [:new, :create]
+    ####
+
+    #### Classifications controller
+    resources :classifications, only: [:create], param: :described_state do
+      get :message, on: :member
```

Comment:
@garethrees this generates the route as `/alaveteli_pro/request/:url_title/classifications/:described_state/message` I'm tempted to switch to `/alaveteli_pro/request/:url_title/classifications/:described_state` using the `new` action - thoughts?

Reply:
Lets keep it as it is for now – I'm not necessarily sure it should be part of this controller in the first place, but IDK. Will look at that with fresh eyes once we've made the more general restructure.